nautobot/tenancy/tests/test_filters.py

```diff
@@ -455,19 +455,6 @@ def test_vrfs(self):
             self.queryset.filter(vrfs__in=vrfs).distinct(),
         )
 
-    def test_has_vrfs(self):
```

Comment:
This and the generic test both failed so I went ahead and removed this one because they're identical. #3213 is open to migrate the rest of the tests covered by the new generic filter tests